cmd/job-trigger-controller-manager/main.go

```diff
@@ -42,6 +43,7 @@ func gatherOptions() (*options, error) {
 	fs.BoolVar(&o.dryRun, "dry-run", true, "Whether to run the controller-manager with dry-run")
 	fs.StringVar(&o.namespace, "namespace", "ci", "In which namespace the operation will take place")
 	fs.Int64Var(&o.jobTriggerWaitInSeconds, "job-trigger-wait-seconds", 60, "Amount of seconds to wait for job to trigger in order to update status")
+	fs.StringVar(&o.dispatcherAddress, "dispatcher-address", "http://prowjob-dispatcher.ci.svc.cluster.local:8080", "Address of prowjob-dispatcher server.")
```

Comment:
It's just food for thought for the future before expanding more. Since we are in the same cluster, and the services need to communicate, I would prefer using gRPC.

Reply:
Do you know of any place where we are doing that already? outside of gangway, of course

Reply:
I am not sure, but in theory we could have a proto repository holding all protobuf configuration that any component can just use based on the service it needs. We could have a list of services with a list of rpc calls that satisfy the needs of our infra. Not directly related to that PR, but in some point I would prefer using gRPC instead of REST API calls.

Reply:
I do think that is a good direction.